websites/B/Bungie/presence.ts

```diff
@@ -0,0 +1,303 @@
+}
+else if (path.includes ("/en/Forums/Post/")){
+  titleFourteen = document.querySelector("#topicPost > div > div.threadMeta > div > div > div.authorMeta > a")
+  titleTwo = document.querySelector("#topicPost > div > div.threadMeta > div > h1")
+  //document.querySelector("#topicPost > div > div.threadMeta > div > h1")
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at: "+titleTwo.innerHTML + "By: " + titleFourteen.innerHTML)
+}
+else if (path.includes ("/en/ClanV2/MyClans")){
+  titleEight = document.querySelector("#clan-container > div > div.container_bodyContent > div > h1")
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at the clans they are apart of")
+}
+else if (path.includes ("/en/ClanV2/Index")){
+  titleSeven = titleSeven = document.querySelector("#mainContent > div.darkThemeContent.grid.full-screen > div > div.container_bodyContent.customScroll > div.header > div.clanIdentity > h1")
+presenceData.startTimestamp = browsingStamp;
+presenceData.details = ("Concerding joining clan "+titleSeven)
+}
+else if (path.includes ("/en/ClanV2/Fireteam")){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Checking out the available fireteams")
+}
+else if (path.includes ("/en/ClanV2/PublicFireteam")){
+  titleNine = document.querySelector("#clan-container > div > div > div > div > div.activity-header > h2")
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Interested in in fireteam "+titleNine.innerHTML)
+}
+else if (path.includes ("en/Groups/SuggestedGroups")){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at the groups Bungie suggested to them")
+}
+else if (path.includes ("en/Groups/MyGroups")){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Searching for groups")
+}
+
+else if (path.includes ("en/Groups/Popular")){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at the groups they're apart of")
+}
+
+else if (path.includes ("/en/Groups/Search")){
+presenceData.startTimestamp = browsingStamp;
+presenceData.details = ("Searching for groups")
+}
+
+else if  (path.includes ("/en/Groups/Chat")){
+  titleTen = document.querySelector("#groupName")
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Interested/Joined group "+titleTen.innerHTML)
+  console.log (titleTen.innerHTML)
+}
+//Creations 
+else if  (path.includes ("/en/Community/Creations")){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at the creations the community made")
+}
+else if  (path.includes ("/en/Community/Detail")){
+ titleEleven = document.querySelector("#mainContent > div.community-detail-header > div > div > div.community-details.flex > div.title")
+  //titleEleven = document.querySelector("#mainContent > div.community-detail-header > div > div > div.community-details.flex > div.title")
```

Comment:
why the comment?

Reply:
Different JavaScript path. Just kept it there just in case it worked out.

Reply:
it is 100% the same tho xD

Reply:
Shhhhhh, don't tell people that! I don't want them knowing my secret...